MeNext/Layout/HostEvent.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Xamarin.Forms;
+namespace MeNext
+{
+	public class HostEvent : ContentPage
+	{
+		public HostEvent()
+		{
+			var layout = new StackLayout
+			{
+				Children = {
+					new Label { Text = "Host Event!\nVarious permissions in a list\n" }
+				}
+			};
+			var permissions = new List<string>();
```

Comment:
We should probably convert this to a type. We probably want descriptions, plain text names, and server permission names.